src/building/house_service.c

```diff
@@ -3,11 +3,12 @@
 #include "building/building.h"
 #include "building/monument.h"
 #include "city/culture.h"
+#include "core/calc.h"
 
 static void decay(unsigned char *value)
 {
     if (*value > 0) {
-        *value = *value - 1;
+        *value = *value - 2;
```

Comment:
This necessary to increase sensitivity. Otherwise decay takes too long (6 months) and difference between a house visited just a tick ago and a house visited two months ago is too small to make a real difference:
		Decay			Months->				
100		1				96	80	64	48	32	16	0

House Type		Pop								
Small Tent		20		480	400	320	240	160	80	0
Large Tent		28		343	286	229	171	114	57	0
Small Shack		36		267	222	178	133	89	44	0
Large Shack		44		218	182	145	109	73	36	0
Small Hovel		52		185	154	123	92	62	31	0
Large Hovel		60		160	133	107	80	53	27	0
Small Casa		68		141	118	94	71	47	24	0
Large Casa		76		126	105	84	63	42	21	0
Small Insula	        76		126	105	84	63	42	21	0
Medium Insula	80		120	100	80	60	40	20	0
Large Insula	        84		114	95	76	57	38	19	0
Grand Insula	        84		114	95	76	57	38	19	0
Small Villa		40		240	200	160	120	80	40	0
Medium Villa	        42		229	190	152	114	76	38	0
Large Villa		90		107	89	71	53	36	18	0
Grand Villa		100		96	80	64	48	32	16	0
Small Palace	        106		91	75	60	45	30	15	0
Medium Palace	112		86	71	57	43	29	14	0
Large Palace	        190		51	42	34	25	17	8	0
Luxury Palace	        200		48	40	32	24	16	8	0

And this is with decay x2:

	Decay		        Months->						
100	2		                96	64	32	0	-32	-64	-96
House Type	Pop								
Small Tent	20		480	320	160	0	-160	-320	-480
Large Tent	28		343	229	114	0	-114	-229	-343
Small Shack	36		267	178	89	0	-89	-178	-267
Large Shack	44		218	145	73	0	-73	-145	-218
Small Hovel	52		185	123	62	0	-62	-123	-185
Large Hovel	60		160	107	53	0	-53	-107	-160
Small Casa	68		141	94	47	0	-47	-94	-141
Large Casa	76		126	84	42	0	-42	-84	-126
Small Insula	76		126	84	42	0	-42	-84	-126
Medium Insul	80		120	80	40	0	-40	-80	-120
Large Insula	84		114	76	38	0	-38	-76	-114
Grand Insula	84		114	76	38	0	-38	-76	-114
Small Villa	40		240	160	80	0	-80	-160	-240
Medium Villa	42		229	152	76	0	-76	-152	-229
Large Villa	90		107	71	36	0	-36	-71	-107
Grand Villa	100		96	64	32	0	-32	-64	-96
Small Palace	106		91	60	30	0	-30	-60	-91
Medium Palac	112		86	57	29	0	-29	-57	-86
Large Palace	190		51	34	17	0	-17	-34	-51
Luxury Palace	200		48	32	16	0	-16	-32	-48

Reply:
Well... sorry for formatting. It is excel worksheet. But you've get the idea.

Reply:
> Well... sorry for formatting. It is excel worksheet. But you've get the idea.

No worries. I updated you comment to disregard formatting, check out how I did it (edit your comment to check) for future reference!

> This necessary to increase sensitivity. Otherwise decay takes too long (6 months) and difference between a house visited just a tick ago and a house visited two months ago is too small to make a real difference:

Is this a consequence of your culture overhaul? I'm not really into the decay rates on C3...

Reply:
It is probably possible to increase model sensitivity by altering calculations and not touching decay rates. But I've played a few missions with x2 decay and the game feels harder and more fun for me personally.

Reply:
We need to be careful here. Even though the decay may feel better suited for you, other players may have trouble adjusting. For now I think we should keep the original cultural decay, and maybe later add a new difficulty option where the decay is higher.

Reply:
Agree. We should be careful. But after introducing Roadblocks player got much more control on where walkers go and default decay x1 is now too long (6 months is ridiculously long period) . The game difficulty is currently off (because of roadblocks).